mediapipe/graphs/instantmotiontracking/subgraphs/calculators/tracked_anchor_manager_calculator.cc

```diff
@@ -26,17 +26,24 @@ constexpr char kAnchorsTag[] = "ANCHORS";
 constexpr char kBoxesInputTag[] = "BOXES";
 constexpr char kBoxesOutputTag[] = "START_POS";
 constexpr char kCancelTag[] = "CANCEL_ID";
-
-// Generates the final model matrix rotations via the IMU (orientation sensors)
-// of the device and user rotation information
+// TODO: Find optimal Height/Width (0.1-0.3)
+constexpr float kBoxHw = 0.2f; // Used to establish tracking box dimensions
```

Comment:
super-nit: actually, let's call this kBoxEdgeSize, just to clarify the Hw part a little